spec/season_spec.rb

```diff
@@ -16,14 +16,14 @@
     }
 
     @stat_tracker = StatTracker.from_csv(locations)
```

Comment:
When set up there was team_data loaded on the Season class, but we needed the game_data.

Same as I did the Season class file itself

Reply:
My branch has game_team_data which has tackles and other data relevant to the Season statistics. I think this should still be initialized with the game_team_data and the team_data to get team names.

Reply:
Ahh sorry, I guess I skipped over that. I have added team_data back into the Season class.